packages/unmock-jsdom/src/persistence/local-storage-persistence.ts

```diff
@@ -48,4 +67,24 @@ export default class BrowserStoragePersistence implements IPersistence {
     return path.join(outdir, ...args);
   }
 
+  private genericSave<T>(hash: string, fn: string, data: T) {
```

Comment:
Use of `T` generics seems like a leftover from `unmock-node`. Is there any reason `any` would not work here? Alternatively, shouldn't the calls in this `persistence` layer then use e.g. `this.genericLoad<IResponseData>(hash, RESPONSE_FILE)`?

Reply:
The generics are inferred, so using `<IResponseData>` is optional.

Reply:
Then it goes the other way around - why are they explicit in `unmock-node`? :sweat_smile: